packages/cells/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@srcbook/cells",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "./dist/index.js",
+  "scripts": {
+    "prebuild": "rm -rf ./dist",
+    "build": "tsc",
+    "dev": "tsc --watch --project .",
+    "lint": "eslint . --max-warnings 0",
+    "check-types": "tsc",
+    "prepublishOnly": "npm run build"
+  },
+  "dependencies": {
+    "clsx": "^2.1.1",
+    "tailwind-merge": "^2.5.2"
+  },
+  "peerDependencies": {
+    "react": "*",
+    "@types/react": "*"
+  }
```

Comment:
I did my best here based on what I've done in the past, but I'm not sure this is right.

1. In theory, the versions of `clsx` and `tailwind-merge` could be different than the versions in `@srcbook/web` - this might be a good property (not having to update everything in lock step means easier migrations), or it might be not (it could lead to duplicate code in the bundle). It also just might not matter right now too 😄 . If the same version is desired, it seems like that might be a place to use that pnpm catalogs feature?

2. In the past with dependency packages like this, I've set react as a peerdependency with little to no explicit version requirement to reduce dependency complexity / cognitive overhead. This is because In the future the hub project would need to be kept up to date and picking too strict of a version range will likely (I've had some experience here) make this difficult. That being said, I could see why such a wide open version range would not be ideal - so, if something more strict would be preferred, maybe the thing to do here is use that pnpm catalogs feature and define a global version of react that could be both used here and for the react version in the web package.

Reply:
Should we also make clsx and tailwind-merge peer deps? I realize that's maybe not ideal in some cases but given we aren't expecting this to be a very generic package (mostly just for our internal use) it seems reasonable?

In either case, I don't think we need to overthink these things too much, at least assuming we don't blow up our asset files with duplicate dependencies

cc @versecafe